### PR TITLE
Introduced "others" translation provider

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Translations from messages.{locale}.xlf were not displayed in back office, now it is. |
| Type? | new feature |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | part of http://forge.prestashop.com/browse/BOOM-1441 |
| How to test? | Test Translations > Others |

> merge https://github.com/PrestaShop/PrestaShop/pull/6353 first, or you cant select "others"

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: src/PrestaShopBundle/Controller/Admin/TranslationsController.php [hunk 1]
```diff
@@ -222,7 +222,7 @@ protected function getTranslationsCatalogue(Request $request)
 
         $locale = $this->langToLocale($lang);
 
-        if (!is_null($theme)) {
+        if (!is_null($theme) && 'themes' === $type) {
```
Comment (Shudrum) [inline src/PrestaShopBundle/Controller/Admin/TranslationsController.php]: Isn't it the same as: https://github.com/PrestaShop/PrestaShop/pull/6353/files ?

**Review by Shudrum — COMMENTED** [review]
Just a single question and all is good :)

File: src/PrestaShopBundle/Controller/Admin/TranslationsController.php (hunk 1 above)
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Controller/Admin/TranslationsController.php]: yes, this is why you have to merge #6353 before, as noticed in PR description. I needed this fix to do the job here.

**Comment by maximebiloe** [comment]
Thanks!

**Pull request merged**